docs/pages/apis/rest.fr.mdx

```diff
@@ -0,0 +1,123 @@
+- `/v1/spells`: Accéder à la liste des sorts.
+- `/v1/spells/{id}`: Accéder à un sort en particulier via son ID.
+
+Les IDs doivent être fournies au format UUID ou en slug
+
+</details>
+
+## Spécifications OpenAPI (OAS)
+
+Pour faciliter l’intégration, nous fournissons un document de [spécification OpenAPI](https://openapis.org) qui décrit les endpoints, les modèles de données et les schémas de requête/réponse de l’API.
+L’API est conforme à la version 3.0.3 de la spécification OAS.
+Vous trouverez notre document OAS actuel [ici](https://api.potterdb.com/v1/openapi.json).
+
+## Format JSON:API
+
+Notre API REST adhère à la [spécification JSON:API](https://jsonapi.org/), qui fournit un moyen cohérent de structurer les requêtes et les réponses de l’API. Voici un aperçu des principales caractéristiques de JSON:API dans le contexte de notre API :
+
+- **Objets** : Chaque ressource est représentée comme un objet avec un type, un `id` et des attributs (`attributes`).
+- **Relations** : Les ressources peuvent être liées à d’autres ressources, par exemple une ressource « livre » (`book`) est liée aux ressources « chapitre » (`chapter`) et vice versa.
+- **Pagination** : Lorsqu’une réponse contient un grand nombre de ressources, la réponse sera paginée. Voir [Pagination](#pagination) pour plus d’informations.
+- **Erreurs** : Les erreurs sont retournées avec des structures normalisées, ce qui facilite la gestion et le dépannage des problèmes.
+
+## Faire des requêtes
+
+Pour faire une demande à l’API REST, vous devez faire une requête `GET` à l’un de nos endpoints disponibles.
+Voici un exemple simple de la façon de récupérer une liste de personnages à l’aide du endpoint `/characters` :
+
+```http
+GET https://api.potterdb.com/v1/characters
+```
+
+Veuillez vous référer au document de spécification OpenAPI pour plus d’informations.
+
+## Pagination
+
+Les réponses avec un grand nombre de ressources seront paginées. La réponse incluera un objet de liens (`links`) avec des liens `first`, `last`, `prev` et `next` (respectivement : premier, dernier, précédent et suivant) pour accéder à la première, la dernière page ainsi qu'aux pages précédentes et suivantes de résultats.
+
+Pour changer le nombre de ressources par page, vous pouvez utiliser le paramètre de requête `page[size]` (la taille maximum - size - étant de 100) comme ceci:
+
+```http
+GET https://api.potterdb.com/v1/characters?page[size]=25
+```
+
+Pour changer de page, vous pouvez utiliser le paramètre de requête `page[number]` ("number" étant le numéro de page souhaité):
+
+```http
+GET https://api.potterdb.com/v1/characters?page[number]=2
+```
+
+## Ransack
+
+Notre API REST supporte une fonctionnalité de filtre avancé et de tri en utilisant [Ransack](https://activerecord-hackery.github.io/ransack/).
+
+### Filtrer
+
+Vous pouvez filtrer une requête en ajoutant le mot-clef `filter` en paramètre de requête. Par exemple, pour filter les personnages par nom, vous utiliserez le prédicat `name_cont` :
```

Comment:
```suggestion
Vous pouvez filtrer une requête en ajoutant le mot-clef `filter` en paramètre de requête. Par exemple, pour filter les personnages par nom, vous utiliserez le prédicat `name_cont`:
```

Reply:
Like for the other suggestions about spaces, I'll wait for your confirmation to apply, or not ;)